creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -261,6 +261,12 @@ ______________________________________________________________________________
 drivers.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* Windows machines need the latest virtio network driver to configure
+the correct MTU on the interface.
+______________________________________________________________________________
```

Comment:
seems unrelated to the purpose of this PR

Reply:
updated the PR name